### DISCUSSION TITLE
RTL support

### DISCUSSION THREAD

**Discussion by iloudsan** [primefaces/community#501] [PrimeNG] [answered]
is there a fast way to make primeng in RTL as i tried to do it with just adding the dir tag it did not work as a lot of css modifications needs to be done in order to achieve this

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've plans in the future to make RTL support, right now we're at the last stage of accessibility. Afterward, we'll implement unstyled mode and pass through props, RTL has a lot of time.
